web3/providers/eth_tester/defaults.py

```diff
@@ -98,26 +128,28 @@ def get_transaction_by_block_hash_and_index(eth_tester, params):
 
 @null_if_indexerror
 @null_if_block_not_found
-def get_transaction_by_block_number_and_index(eth_tester, params):
+def get_transaction_by_block_number_and_index(
+    eth_tester: "EthereumTester", params: Any
+) -> TxReceipt:
     block_number, transaction_index = params
     block = eth_tester.get_block_by_number(block_number, full_transactions=True)
     transaction = block['transactions'][transaction_index]
     return transaction
 
 
-def create_log_filter(eth_tester, params):
+def create_log_filter(eth_tester: "EthereumTester", params: Any) -> int:
     filter_params = params[0]
     filter_id = eth_tester.create_log_filter(**filter_params)
     return filter_id
 
 
-def get_logs(eth_tester, params):
+def get_logs(eth_tester: "EthereumTester", params: Any) -> List[LogReceipt]:
```

Comment:
For each of these that is a defined API we should know what the `params` value looks like and thus be able to use stricture types than `Any`.

Reply:
👍 Will take care of tightening up the params value in a different pr to avoid bloat.